src/middlewared/middlewared/etc_files/netdata/netdata.conf.mako

```diff
@@ -38,7 +38,7 @@
 [plugins]
 	proc = yes
 	diskspace = no
-	cgroups = no
+	cgroups = yes # Useful stats (CPU, MEM, DISK) for VMs and systemd services
```

Comment:
Also this won't be exposed to the end consumer i.e netdata will start collecting more metrics but reporting service won't be returning any of that data.

Can i take over this if we just want to enable this so i can update the disk usage method as well to account for the change ?

Reply:
We dont need it to be exposed in our UI. This will be available in the netdata UI and thats enough for the time being.

Yes, please feel free to take it over for disk space accounting.